collectr/webfront/views.py

```diff
@@ -162,7 +162,7 @@ def get_queryset(self):
             collection = 'all'
 
         return queryset.filter(
-            collection=self.user_collections.get(name__iexact=collection)
+            collection=self.user_collections.filter(name__iexact=collection)
```

Comment:
Hum well, the collection should exists, if it doesn't, there is a big problem no ?